crates/egui-wgpu/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ All notable changes to the `egui-wgpu` integration will be noted in this file.
 * `Renderer::update_texture` no longer creates a new `wgpu::Sampler` with every new texture ([#2198](https://github.com/emilk/egui/pull/2198))
 * `Painter`'s instance/device/adapter/surface creation is now configurable via `WgpuConfiguration` ([#2207](https://github.com/emilk/egui/pull/2207))
 * Fix panic on using a depth buffer ([#2316](https://github.com/emilk/egui/pull/2316))
+* Fix panic if we can't find a device ([#2428](https://github.com/emilk/egui/pull/2428))
```

Comment:
Move to the `Unreleased` section please!

Reply:
Fixed